### PR TITLE
Update EnhancedNavBall.netkan

### PR DESCRIPTION
**Pull request by politas**
Has been tested and is working with 1.0.5

### CODE REVIEW

**Comment by plague006** [comment]
We don't hardcode ksp_versions to .netkans unless absolutely necessary, because it will create a situation in the future where our old hardcoded values will overwrite values scraped from the KerbalStuff page. Please use a version-specific override. For an example see https://github.com/KSP-CKAN/NetKAN/blob/master/NetKAN/Achievements.netkan

**Comment by politas** [comment]
Thanks. I think I saw that netkan_override stuff in one mod, but wasn't sure what it was about.

**Comment by plague006** [comment]
Lines 5+6 should be removed (no hardcoded ksp_versions in the main metadata).
Line 7: `"x_netkan_override" : [` has an opening square bracket but there's no associated closing bracket on line 15/16.

**Comment by politas** [comment]
Ah, that makes more sense. Fixed.

**Comment by politas** [comment]
Jenkins seems to be confused.

File: NetKAN/EnhancedNavBall.netkan [hunk 1]
```diff
@@ -2,5 +2,17 @@
...
+	[
+		{
+			"version" : "1.3.6.0",
+			"delete" : [ "ksp_version" ],
+			"override" : {
+				"ksp_version_min" : "1.0.2",
+				"ksp_version_max" : "1.0.5"
+			}
+		}
+	]
```
Comment (plague006) [inline NetKAN/EnhancedNavBall.netkan]: needs comma

**Comment by politas** [comment]
D'oh!

**Pull request merged**